todo/src/App.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import {connect} from "react-redux"
+import './App.css';
+import IndividualTodo from "./components/IndividualTodo"
+import {addinput} from "./actions/action"
+
+
+class App extends Component {
+  state = {
+    inputMember :  "",
+    error: ""
+  }
+
+  
+  changeHandler = e =>{
+    e.preventDefault();
+    this.setState ({
+      inputMember: e.target.value
+    })
+  
+    console.log(this.state.inputMember)
+  }
+
+
+  addMember = e=>{
+    e.preventDefault()
+    if(this.state.inputMember !== ""){
+    this.props.addinput(this.state.inputMember.trim())
+    }
+    this.setState({
+      inputMember : ""
+    })
+  }
+
+
+  render() {
+    console.log(this.props.todoList)
+    console.log(this.state.error)
+ 
+    return (
+      <div className="App">
+        
+          <h3> A title </h3>
+          <input type="text" 
+                  value={this.state.inputMember}
+                 name="inputMember"
+                onChange={this.changeHandler} />
+                <button onClick={this.addMember}> Add a value </button>
```

Comment:
It's usually a better user experience to wrap your input and button inside a form, then place your submit handler (in this case addMember) in an onSubmit on the form, rather than on onClick on the button. This way the user can add their task either by pressing enter or by clicking the button.